core/controllers/image/controller.php

```diff
@@ -3,12 +3,64 @@
 
 // api style controller - end output
 ob_end_clean();
-
+ob_end_clean(); // not a mistake, a safety net
 // router
 
 $segments = CMS::Instance()->uri_segments;
 $segsize = sizeof($segments);
 
+// handle list api request
+if ($segments[1]=='list_images') {
+	$mimetypes = array_filter(explode(',',Input::getvar('mimetypes','STRING'))) ?? null;
```

Comment:
null coalesce (??) input function in explode, since if its null, in php 8 it will explode